src/components/EventCell/styles.ts

```diff
@@ -1,19 +1,33 @@
 import { StyleSheet } from 'react-native';
-import { COPPER_GREY, WHITE, COPPER } from '../../styles/colors';
+import { COPPER_GREY } from '../../styles/colors';
 import { FIRA_SANS_BOLD, FIRA_SANS_REGULAR } from '../../styles/fonts';
 import { BORDER_RADIUS, BORDER_WIDTH, MARGIN, PADDING } from '../../styles/metrics';
 
 const LEFT_BORDER_CELL = BORDER_WIDTH * 8;
 
-export default StyleSheet.create({
+type eventCellType = {
+  borderColor: string,
+  backgroundColor: string,
+};
+
+export type eventCellStyleType = {
+  cell: object,
+  infoContainer: object,
+  eventTitle: object,
+  eventInfo: object,
+  timeContainer: object,
+  iconContainer: object,
+};
+
+const eventCellStyle = ({ borderColor, backgroundColor } : eventCellType) => StyleSheet.create({
   cell: {
     display: 'flex',
     flexDirection: 'row',
     borderRadius: BORDER_RADIUS.MD,
     borderWidth: BORDER_WIDTH,
     borderLeftWidth: LEFT_BORDER_CELL,
-    borderColor: COPPER[100],
-    backgroundColor: WHITE,
+    borderColor,
+    backgroundColor,
     marginHorizontal: MARGIN.MD,
     paddingHorizontal: PADDING.LG,
   },
```

Comment:
cf [ma veille là dessus ](https://alenvi.slack.com/archives/CK0T9HMK9/p1646405168707789), on peut faire des styles dynamiques plus facilement. 
Et ça t'évite la galère de `eventCellStyleType` en plus je pense.

Que penses-tu, d'essayer ça ici ?

Reply:
J'ai essayé mais typescript renvoie une erreur:  `This expression is not callable. No constituent of type 'ViewStyle | TextStyle | ImageStyle' is callable`
L'erreur a l'air connue : [](https://stackoverflow.com/questions/71001638/how-to-solve-viewstyle-typescript-error-in-react-native)
Une autre manière de faire est de creer une methode qui renvoie un objet de type ViewStyle comme j'ai fait, qu'en penses-tu ? 

J'aime bien cette methode car ca évite de créer dans le composant un state qui gère le style et le style reste assez simple.